lua/core/state.lua

```diff
@@ -4,23 +4,23 @@
 state = {}
 state.tape = 0
 state.script = ''
-state.path = _path["dust"]
-state.data = _path["dust"]
+state.path = _path.code
+state.data = _path.data
 state.name = ''
 state.shortname = ''
 state.clean_shutdown = false
 
 --- read state.lua and set parameters back to stored vals.
 state.resume = function()
   -- restore mix state
-  mix:read(_path["dust"].."system.pset")
+  mix:read(_path.data.."system.pset")
```

Comment:
Could very well be I'm misunderstanding something, but wouldn't it make more sense to use `state.data` here instead of `_path.data`?

Reply:
both would work - but i agree that minimizing the use of globals established elsewhere is a good thing

Reply:
they are not identical.

`state.data` is dynamic, it's the user data folder for the script ie `/dust/data/awake/`

whereas `_path.data` is always `/dust/data/`